### DISCUSSION TITLE
Getting charged by Tuya

### DISCUSSION THREAD

**Discussion by joe-sydney** [tuya/tuya-home-assistant#454] [General]
Hi there, I got charged today for using the Home Assistant integration. I think I might have exceeded quotas, but I just have 2 devices, both of them being energy monitoring plugs so sending energy data regularly.

The following was in the bill details:

API CALL   FOREIGN TRIAL数量 | API CALL FOREIGN   TRIAL费用
1549 | 3.3

Is this to be expected? It won't really be sustainable if data from energy monitoring plugs are going to drive this cost.

Cheers, Joe

**Comment by bh56** [comment]
Any luck on finding out about charges.  Did you stop using or did this stop.  It's not clear to me on the "Trial", if you can keep that going or if eventually you have to start paying to be able to integrate to home assitant.

**Reply by joe-sydney** [reply]
The charges were automatically reversed, it may have just been a glitch.

**Comment by bh56** [comment]
Thanks Joe.  Can you clarify for me, in order to configure the home assistant/Tuya integration, you need to create an account.  From the documentation it appears you set up initially trial account.  But what I'm not clear from the FAQ and on Tuya site, is if the trial is limited and you have to step up to a paid subscription at that point.  Any further insight on this would be appreciated.
